package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "todo",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
```

Comment:
- does this use turbopack by default or should we add the `--turbopack` flag?

Reply:
Has to be added explicitly. Missed by me. 
Added now. 
Ref doc:- https://nextjs.org/docs/architecture/turbopack